lib/Elastica/Search.php

```diff
@@ -22,6 +22,7 @@ class Search
     const OPTION_SIZE = 'size';
     const OPTION_SCROLL = 'scroll';
     const OPTION_SCROLL_ID = 'scroll_id';
+    const OPTION_QUERY_CACHE = 'query_cache';
```

Comment:
Thanks, fixed. I'll have a look how to run the tests but if it takes more time I'm not sure when I can get to that.